db/base/models.py

```diff
@@ -58,6 +58,19 @@ class Satellite(models.Model):
                               help_text='Ideally: 250x250')
     tle1 = models.CharField(max_length=200, blank=True)
     tle2 = models.CharField(max_length=200, blank=True)
+    ALIVE = 'a'
+    DEAD = 'd'
+    RE_ENTERED = 'r'
+    STATUS = (
+        (ALIVE, 'Alive'),
+        (DEAD, 'Dead'),
+        (RE_ENTERED, 'Re-entered'),
+    )
+    status = models.CharField(
```

Comment:
I suggest you use the same code convetions with the rest of the code. See the `source` field on the `DemodData` model and the `DATA_SOURCES` variable. So place the `STATUS` variable on top (maybe name it something like `SATELLITE_STATUS` to make more sense) and try to keep the model code as simple as possible.